<?xml version="1.0"?>
<launch>

    <arg name="rviz" default="true" />
    <arg name="project" default="multi_sensor_mapping"/>

    <!-- Lidar IMU Mapper -->
    <node pkg="$(arg project)" type="reflection_column_viewer" name="reflection_column_viewer" output="screen" respawn="false">
        <param name="bag_path" value="/home/hkw/DATA/Dataset/Hunter/Shihushan-ReflectiveColumn-01.bag"/>
        <param name="lidar_topic" value="/rslidar_points"/>
    </node>


    <!--- Rviz -->
    <!-- <group if="$(arg rviz)">
      <node pkg="rviz" type="rviz" name="$(arg project)_rviz" args="-d $(find multi_sensor_mapping)/rviz/lidar_mapper.rviz" />
    </group> -->
</launch>
